packages/flutter/lib/src/material/scaffold.dart

```diff
@@ -36,19 +37,94 @@ enum _ScaffoldSlot {
   statusBar,
 }
 
+/// A position that the [FloatingActionButton] can dock to in a [Scaffold].
+/// 
+/// [getOffset] 
```

Comment:
(not sure if you want detailed review comments yet, but if you do: looks like these docs are incomplete)

Reply:
Documentation is ready.